guides/cookbook.md

```diff
@@ -1632,6 +1906,55 @@ class CsvToIon
 ```
 </div>
 
+<div class="tabpane Go" markdown="1">
+
+```Go
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"strconv"
+	"strings"
+	
+	"github.com/amzn/ion-go/ion"
+)
+
+func main() {
+	file, er := os.Open("c://values.cvs")
+	if er != nil {
+		panic(er)
+	}
+	defer file.Close()
+
+	buf := strings.Builder{}
+	writer := ion.NewTextWriter(&buf)
+
+	scanner := bufio.NewScanner(file)
+	scanner.Scan() // to skip the first row (header line)
```

Comment:
Is there context on why we're skipping the first row here?

Reply:
The contents of the file are:
```
 id,type,state
 1,foo,false
 2,bar,true
 3,baz,true
```
And is explained in that section.
We are skipping the first line as it is the header and just have the titles